docs/features/workspaces.md

```diff
@@ -25,22 +25,21 @@ By default, all workspaces are public. This means that anyone with the link to y
 
 The expandable sidebar includes:
 
- - Options for creating and viewing workspaces.
- - Access to your Contributor and Repository Insights pages.
- - Links to support.
+- Options for creating and viewing workspaces.
+- Access to your Contributor and Repository Insights pages.
+- Links to support.
 
- ![Sidebar](../../static/img/workspace-sidebar.png)
+![Sidebar](../../static/img/workspace-sidebar.png)
 
-:::tip
+:::info
 
- We recently had a naming change. The Contributors Insights section was previously called [Lists](contributor-insights.md), and the Repositories Insights section was previously called [Insights](./repo-insights.md).
+We recently had a naming change. The [Contributors Insights](contributor-insights.md) section was previously called Lists, and the [Repositories Insights](./repo-insights.md) section was previously called Insights.
```

Comment:
Do we need to mention "section" here as in Contributors Insights section?
I'm thinking as we're talking about features, we can omit the word "section". Or, change it with "feature".

```suggestion
We recently had a naming change. The [Contributors Insights](contributor-insights.md) feature was previously called Lists, and the [Repositories Insights](./repo-insights.md) feature was previously called Insights.
```